src/suspendplugin.c

```diff
@@ -384,6 +441,14 @@ suspend_handle_display_off()
     if (access(WMTWIFI_DEVICE, F_OK) == 0) {
         // only suspend/unsuspend if we are not in ap mode
         if ((if_nametoindex("ap0")) == 0) {
+            /*
+                wmtWifi (with gen3 driver at least) in Android requires to set power_save
+                to `off` and after that power_save to `on` to correctly suspend wmtWifi device.
+                We are doing that just before calling suspend_set_wmtwifi() after display is turned off.
+                This solution doesn't give us any disadvantages in comparison with Android behaviour.
+            */
+            (void) suspend_set_powersave("wlan0", FALSE);
+            (void) suspend_set_powersave("wlan0", TRUE);
```

Comment:
Does it make sense to declare the function with a return value but ignore the return value every time it is being used?

Reply:
I'm sure that return value could be used later by us or other people. Just `reserved` return value.

Reply:
I'm not so sure about that :) Besides, it's easy enough to add when and if it becomes needed. I suggest to stick to "do not introduce new entities without necessity" principle. And this `(void)` thing isn't necessary too, it's just littering  the code IMO. Not that it's a big deal but if it were the code that I'm maintaining, I would take it out.